docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -422,6 +422,13 @@ fields:
   - Birthdate: x.birthdate
     datatype: BirthDate
 ---
+id: you birthdate question
+question: |
+  When were your born?
```

Comment:
```suggestion
  When were you born?
```

Reply:
As we add more questions with direct address, we should really work on #109. But I do think this is a safe change in keeping with our other direct address questions